### ISSUE TITLE
Code Lens not showing up for individual test cases. .class

### ISSUE THREAD

**Issue by MxD-js** [microsoft/vscode-java-test#1196]
Hi, 

I have a tests.jar package that has the tests included, I did not write the tests, instead the've been downloaded by maven, the pom.xml has them as dependency.  I've downloaded the sources as well, however Code Lens does not provide the Run Test| Debug Test above each test case, or the test class. 

Here's the video, I have a TestSuite.java that shows the Run Test | Debug Test, but when I navigate to the Broker.class test, the Code Lens does not show the individual Run Test | Debug Test above each test, like shown here. 
https://code.visualstudio.com/docs/java/java-testing#_rundebug-test-cases

https://user-images.githubusercontent.com/60336163/117559253-8f135c80-b038-11eb-8a51-e1720ba4b9c1.mov

video if it in action:

 Running vsCode 1.56 w/ all extensions latest version.

**Comment by MxD-js** [comment]
precisely. see screen shot below this is after sources downloaded. The green play buttons starts the test for each individual test case

![image](https://user-images.githubusercontent.com/60336163/117772733-587e4300-b1ec-11eb-9543-94fb1b266bd2.png)

**Comment by jdneo** [comment]
Ok, so far the implementation is only register the code lens on `*.java` files. So it is as expected.

I think the behavior of showing the content of the BrokerTest class in VS Code and IntelliJ are very similar. Only the file extension names are different from `.java` and `.class`.

So the question is whether we need to also apply it to `*.class` files.

**Comment by jdneo** [comment]
Unfortunately, we haven't expose this setting for now. But this could be a potential way to support this request.
